From 95111d534cce3d1cecebace25d5dc6dc11800150 Mon Sep 17 00:00:00 2001
From: David Wagner <david.wagner@intel.com>
Date: Fri, 19 Apr 2013 14:59:10 +0200
Subject: [PDK MR2 Temp] Revert "[DO NOT UPSTREAM][Background Music] Support audio playback"

BZ: 117199

This patch breaks the compilation because
e8081ae0e1981d7ea4b9e3a6d4b8a93e9409b0bd
is missing in system/core

This reverts commit 2a1ff53f3b252f4669b6e86b85dedeaee22b44c5.

Change-Id: I51411af0ad7241200004403844b55480cfbf1e21
---
 audio_policy/AudioPolicyManagerALSA.cpp |   10 ++++------
 1 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/audio_policy/AudioPolicyManagerALSA.cpp b/audio_policy/AudioPolicyManagerALSA.cpp
index d49c09f..00d226c 100644
--- a/audio_policy/AudioPolicyManagerALSA.cpp
+++ b/audio_policy/AudioPolicyManagerALSA.cpp
@@ -356,7 +356,6 @@ audio_devices_t AudioPolicyManagerALSA::getDeviceForStrategy(routing_strategy st
     uint32_t device = 0;
 
     device = baseClass::getDeviceForStrategy(strategy, fromCache);
-
     AudioOutputDescriptor *hwOutputDesc = mOutputs.valueFor(mPrimaryOutput);
     uint32_t currentDevice = (uint32_t)hwOutputDesc->device();
 
@@ -396,17 +395,16 @@ audio_devices_t AudioPolicyManagerALSA::getDeviceForStrategy(routing_strategy st
         case STRATEGY_MEDIA:
             // We want to supersede the default policy when we are in call, so that we can change the volume
             // settings for MEDIA streams when in call, on BT SCO, SPEAKER or EARPIECE.
-            if (isInCall()) {
+            if (isInCall()){
                 // Retrieve the device in use for the call
                 AudioOutputDescriptor *hwOutputDesc = mOutputs.valueFor(mPrimaryOutput);
                 uint32_t currentDevice = (uint32_t)hwOutputDesc->device();
-                if (((currentDevice == AudioSystem::DEVICE_OUT_BLUETOOTH_SCO_CARKIT)
+                if ((currentDevice == AudioSystem::DEVICE_OUT_BLUETOOTH_SCO_CARKIT)
                     || (currentDevice == AudioSystem::DEVICE_OUT_BLUETOOTH_SCO_HEADSET)
                     || (currentDevice == AudioSystem::DEVICE_OUT_BLUETOOTH_SCO)
                     || (currentDevice == AudioSystem::DEVICE_OUT_SPEAKER)
-                    || (currentDevice == AudioSystem::DEVICE_OUT_EARPIECE))
-                    && ((device != AUDIO_DEVICE_OUT_REMOTE_BGM_SINK))) {
-                    ALOGD("%s- superseding STRATEGY_MEDIA policy while in call from device 0x%x to 0x%x", __FUNCTION__, device, currentDevice);
+                    || (currentDevice == AudioSystem::DEVICE_OUT_EARPIECE)){
+                    ALOGV("%s- superseding STRATEGY_MEDIA policy while in call from device 0x%x to 0x%x", __FUNCTION__, device, currentDevice);
                     device = currentDevice;
                 }
             }
-- 
1.7.4.1

